uapi/index.mjs

```
@@ -53,6 +53,17 @@ export class UAPI {
     }
   }
 
+  async setRegion(region) {
+    const regionUrl = this.getDomain(region);
+    try {
+      this.CRM.setUrl(regionUrl);
+      this.Commerce.setUrl(regionUrl);
+      this.Accounting.setUrl(regionUrl);
+      this.Webhooks.setUrl(regionUrl);
+      this.url = regionUrl;
+    } catch (error) {}
```

Do we really want to swallow all errors?